### PR TITLE
File uploads to the review project

### PR DESCRIPTION
**Pull request by Obimbo07**
This is an update on the index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -141,7 +141,7 @@ <h2 class="card-title">Profesional Art Printing Data</h2>
       </section>
       <section class="about">
         <div class="about-me">
-          <h2 class="about-title">About me</h2>
+          <h2 class="about-title">About</h2>
```
Comment (Obimbo07) [inline index.html:144]: Please change the content of the About to the one used in Figma design

File: index.html [hunk 2]
```diff
@@ -141,7 +141,7 @@ <h2 class="card-title">Profesional Art Printing Data</h2>
...
           <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
           <button class="resume">Get my resume</button>
         </div>
```
Comment (Obimbo07) [inline index.html:147]: Use the <button> tag outside the text container. We recommend that for proper CSS

Comment (Obimbo07) [inline index.html:147]: Use the <button> tag outside the text container. We recommend that for proper CSS

**Review by Obimbo07 — COMMENTED** [review]
#Require Changes♻️

Please check my comments below and make all  required changes.

Submit the project for a re-review when done.

Thanks🤗

File: index.html (hunk 1 above)
Comment (koachcode) [reply index.html:144]: - [Optional] Please always try to give a reason for the required change.

**Review by koachcode — APPROVED** [review]
Hi @Obimbo07 , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Good use of GitHub inline comment  :100: 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**